### PR TITLE
Remove wrapped_clang tpl workaround files

### PR DESCRIPTION
**Pull request by keith**
Per the comment that was here, these shouldn't be needed since there
shouldn't be issues with building wrapped_clang.

### CODE REVIEW

File: tools/cpp/osx_cc_configure.bzl [hunk 1]
```diff
@@ -120,30 +118,17 @@ def configure_osx_toolchain(repository_ctx, overriden_tools):
...
-                  error_msg)
-            repository_ctx.symlink(
-                paths["@bazel_tools//tools/osx/crosstool:wrapped_clang.tpl"],
-                "wrapped_clang",
-            )
-            repository_ctx.symlink(
-                paths["@bazel_tools//tools/osx/crosstool:wrapped_clang_pp.tpl"],
-                "wrapped_clang_pp",
-            )
+            fail("wrapped_clang failed to generate. This shouldn't cause " +
```
Comment (sergiocampama) [inline tools/cpp/osx_cc_configure.bzl]: Remove the "this shouldn't cause problems" part, since it will cause a problem now :)

Comment (keith) [reply tools/cpp/osx_cc_configure.bzl]: Ha yea done, left the part about filing an issue

**Comment by keith** [comment]
The windows failure her appears to be the same as the one on master right now

**Comment by keith** [comment]
https://github.com/bazelbuild/bazel/issues/7515

**Comment by keith** [comment]
@aragos can you review this one as well if that makes sense?

**Comment by keith** [comment]
Thanks!

**Pull request closed without merging**